test/unittests/utils/test_dwi.py

```diff
@@ -53,3 +56,221 @@ def test_rename_files(tmp_path):
     ).exists()
     assert (tmp_path / "foo/bar/baz.nii.gz").exists()
     assert (tmp_path / "foo/bar/sub-02_ses-M666_dwi_foo.nii.gz").exists()
+
+
+def test_get_b0_filter_error(tmp_path):
+    from clinica.utils.dwi import get_b0_filter
+
+    with pytest.raises(
+        FileNotFoundError,
+        match="File not found",
+    ):
+        get_b0_filter(tmp_path / "foo.bval")
+
+
+def test_get_b0_filter(tmp_path):
+    from clinica.utils.dwi import get_b0_filter
```

Comment:
this looks like it can be factorized using parametrize

Reply:
That's a very good point, thanks for the suggestion. 👍 
Implemented in [8f5d89d](https://github.com/aramis-lab/clinica/pull/883/commits/8f5d89dfe18039025d91df1b8e0d9bef5e0c9be9)